.circleci/config.yml

```diff
@@ -0,0 +1,71 @@
+version: 2.1
+
+executors:
+  py38:
+    docker:
+      - image: circleci/python:3.8
+  py37:
+    docker:
+      - image: circleci/python:3.7
+  py36:
+    docker:
+      - image: circleci/python:3.6
+
+commands:
+  setup_requirements:
+    description: Setup requirements
+    parameters:
+      python_version:
+        type: string
+    steps:
+      - checkout
+      - run: |
+          sudo chown -R circleci:circleci /usr/local/bin
+          sudo chown -R circleci:circleci /usr/local/lib/<< parameters.python_version >>/site-packages
+      - run:
+          name: Install pip setuptools
+          command: python -m pip install -U "pip==21.0.1" "setuptools>=38.0" wheel
+      - run:
+          command: |
+            sudo pip install -r test_requirements.txt
+  e2e:
+    description: Run end to end tests for all starters
+    steps:
+      - run:
+          command: behave features
+
+jobs:
+  security_scan:
+    executor: py38
+    steps:
+      - setup_requirements:
+          python_version: "python3.8"
+      - run:
+          command: bandit -ll -r .
+  e2e_38:
+    executor: py38
+    steps:
+      - setup_requirements:
+          python_version: "python3.8"
+      - e2e
+  e2e_37:
+    executor: py37
+    steps:
+      - setup_requirements:
+          python_version: "python3.7"
+      - e2e
+  e2e_36:
+    executor: py36
+    steps:
+      - setup_requirements:
+          python_version: "python3.6"
+      - e2e
+
+workflows:
+  version: 2
+  build:
+    jobs:
+      - security_scan
+      - e2e_38
```

Comment:
Should these tests be run on Windows as well?

Reply:
Ah that's a good point.

Reply:
Getting windows to work was really fiddly so I'll defer it to a separate ticket, and reap the benefits of having _some_ CI working in the meantime.

Reply:
No worries, that sounds good!